beetsplug/mpdstats.py

```diff
@@ -203,6 +203,9 @@ def update_item(self, item, attribute, value=None, increment=None):
     def update_rating(self, item, skipped):
         """Update the rating for a beets item.
```

Comment:
Can you add a note to the docstring here? Something like

    `item` can either be a beets `Item` or None, in which case nothing changes